spec/models/miq_schedule_spec.rb

```diff
@@ -119,6 +165,15 @@
         expect(miq_schedule_array['filter']).to be_kind_of(MiqExpression)
         expect(miq_schedule_array['resource_type']).to eq("AutomationRequest")
       end
+
+      it "imports schedule" do
+        miq_schedule_array = MiqSchedule.export_to_array([miq_schedule.id], MiqSchedule).first["MiqSchedule"]
+        imported_schedule = MiqSchedule.import_from_hash(miq_schedule_array).first
+
+        expect(imported_schedule.sched_action[:method]).to eq("run_report")
+        expect(imported_schedule.userid).to eq(user.userid)
+        expect(imported_schedule.resource_type).to eq("MiqReport")
```

Comment:
similar like here https://github.com/ManageIQ/manageiq/pull/19257/files#r328638603

We are not using MiqReport in report_type for AutomationTask but I think `Vm`,..